src/include/daos/drpc.pb-c.h

```diff
@@ -60,16 +94,22 @@ struct  _Drpc__Call
 
 /*
  **
- * Response is the data to the Call with a given sequence number.
- * sequence is the sequence number of the drpc call for the response.
- * status represents the return/faulure value of the call
- * body represents the returned data if a call returns more than just a status.
+ * Response describes the result of a dRPC call.
  */
 struct  _Drpc__Response
 {
   ProtobufCMessage base;
+  /*
+   * Sequence number of the Call that triggered this response.
+   */
   int64_t sequence;
+  /*
+   * High-level status of the RPC. If SUCCESS, method-specific status may be included in the body.
```

Comment:
(style)  line over 80 characters

Reply:
Checkpatch should be ignoring generated files. Will add this one to Jenkinsfile.